blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -40,6 +42,12 @@ ALTER TABLE can be used to alter an existing table.
 While columns can be added at any time, adding a new
 :ref:`generated column <ref-generated-columns>` is only possible if the table is empty.
 
+``OPEN`` and ``CLOSE`` can be used to open or close the table, respectively.
+Closing a table prevents all operations, other than ``ALTER TABLE ... OPEN`` to
+fail. Closing a single partition within a partitioned table will also prevent all
```

Comment:
This is a scenario @seut and I just discussed - we decided to throw an exception when building the route if a where results in a partition specific lookup, and that partition is not open. See 7513adb2c76437a345642820782d12b5c4b3e26d